plugins/memcached/scripts/clean.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
```

Comment:
How user get the scripts? Install plugin or fetch tarball by network? 
And the path of script is also important for playbook execute.

Reply:
I think maybe we can provide a center repo, like docker hub or , user can contribute all kind of scripts for us or download them to use?
Actually the playbook can do anthing, like maintain machines or something curveadm don't support. If we gather them, it will become powerful, it's an interesting thing :)

Reply:
Great idea dude!
If we can provide a central repository, then `corveadm playbook` can add a function `palybook pull xxx` to download the corresponding xxx from the central repository.